x-pack/plugins/observability/public/hooks/slo/helpers/convert_error_for_use_in_toast.ts

```diff
@@ -0,0 +1,15 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+export function convertErrorForUseInToast(error: Error) {
+  const newError = {
+    ...error,
+    message: Object(error).body.message,
```

Comment:
Should we be more defensive here and fallback to a generic message if the body.message does not exist?

Reply:
Can be shored up for sure